.meta/mkcompletion.py

```diff
@@ -6,13 +6,48 @@
 import tqdm.cli
 from io import open as io_open
 from os import path
+import re
+
+def doc2opt(doc):
+    """
+    doc  : str, document to parse
+    """
+    options = ['--' + i.strip('  :') for i in re.findall(r'\w+  :', doc)]
+    return options
+
+# CLI options
+options = ['-h', '--help', '-v', '--version']
+options += doc2opt(tqdm.tqdm.__init__.__doc__)
+options += doc2opt(tqdm.cli.CLI_EXTRA_DOC)
+
+# Remove options without CLI support
+no_support = ['--' + i for i in tqdm.cli.UNSUPPORTED_OPTS]
+# TODO: Check and remove option `name`
```

Comment:
Option `name` present in the docstring and not present in the arguments. Use case not defined in the docs. [cli.py L101](https://github.com/tqdm/tqdm/blob/master/tqdm/cli.py#L101)
For now, it has been manually removed. More information required on this.

Reply:
Yeah as the comment says, no idea why it's needed - just a hack to keep the parser happy. Shouldn't be exposed and not part of any completion.